### ISSUE TITLE
longer than one page

### ISSUE THREAD

**Issue by MiLeung** [amazingandyyy/resume#1]
Only have a resume longer than one page if you have at least 10 years of experience. I heard that's a thing.

**Comment by amazingandyyy** [comment]
@MiLeung  yeah, I also heard that. But which part you think I should remove from this resume and which part you think I can make it more compact? Thanks man

**Comment by MiLeung** [comment]
I would remove any hackathon/side projects unless it's something important/big like maintaining a library or you're trying to turn it into a startup.

You could try a different layout. I use https://content.creddle.io for this and they also automatically change the font size of things to make everything fit on one page.

This just a personal opinion of mine, but I try to remove anything that is implicitly shown through the stuff that is on your resume. So I don't list skills like react, or javascript because my work experience already shows that I know js and react.

**Comment by amazingandyyy** [comment]
@MiLeung Cool, good to know. btw I like [Feross's] (https://feross.org/resume/) :)
